tests/unit/client.py

```diff
@@ -485,6 +540,7 @@ def test_port_delete_error(self):
 
     def test_port_connect_nic(self):
         C.port.register('mock-01', 'gi1/1/5')
+        C.node.add_nic('node-08', 'eth0', 'aa:bb:cc:dd:ee:ff')
```

Comment:
Not sure I understand why you added this (and tweaked a few other functional parts of the tests)? Perhaps this is related to the travis failures?